src/game_api/script/usertypes/screen_lua.cpp

```diff
@@ -107,6 +107,10 @@ void register_usertypes(sol::state& lua)
         &ScreenTitle::particle_torchflame_flames_animated,
         "particle_torchflame_ash",
         &ScreenTitle::particle_torchflame_ash,
+        "background_music",
```

Comment:
Nice, but I'll just nitpick again that we now have a "backgroundmusic", "background_music" and "bg_music" exposed, and isn't all music background music, so maybe just call all of them "music"?

Reply:
Like this? or you mean the struct name as well?

Reply:
The usertype names are fine.